Add better docs to project (#36)
ka-tautoro-2020/coject: README.md

npm run dev
```

If you see warning `API key does not start with "SG.".`, make sure you've put a real SendGrid API key in `.env`. Ask your teammates for more info.

To run in production:

```sh
npm start
```

## JS Docs/ Better-Docs

---

> 'JS Doc' is used to document this project.
> 
> We would like to keep the project documentation up to date with changes to the main branch. For a pull request to be approved all functions should include a 'JS doc' block.

Reviewer: ```suggestion
JSDoc is used to document this project.

We would like to keep the project documentation up to date with changes to the main branch. For a pull request to be approved all functions should include a JSDoc block.
```

💭 For consistency with the rest of the documentation, and also matches how it's spelt on the JSDoc website.

example:

```

>     /**

Reviewer: ```suggestion
/**
```

 * Add userID and EventId to users_projects table
 * @param {string} userId - User Id
 * @param {string} eventId - eventId
 *
 * @returns {Promise.<number[]>} array[0] team id
 */
 const function joinEvent(userId, eventID){...}
```

> Project documentation is found in the [docs]('./docs') folder.

Reviewer: ```suggestion
Project documentation is found in the [docs](./docs) folder.
```
💭 Markdown will include the quotes in the URL.

More information about JSdoc and creating doc blocks can be found in the [JSdoc documentation](https://jsdoc.app/index.html)

> A JSdoc quick reference/cheatsheet found here: [Devhints JSdoc](https://devhints.io/jsdoc)

Reviewer: 👏🏻 Good find!
